src/client/apps/articles_list/test/routes.test.ts

```diff
@@ -1,3 +1,7 @@
+/**
+ * @jest-environment node
+ */
+
```

Comment:
The few files where I've set the `jest` environment to `node` are a result of `js-dom` not having a `TextEncoder` function, which caused `mongodb` to break on a `TypeError`. This was a tricky one to debug but I found a anser that pointed to the root of the cause in a comment from a `jest` maintainer, [here](https://github.com/jsdom/jsdom/issues/2524#issuecomment-902027138). 

Please tell me if you know of any reason this should not be done.